udf-compiler/src/main/scala/com/nvidia/spark/udf/Plugin.scala

```diff
@@ -49,7 +51,7 @@ case class LogicalPlanRules() extends Rule[LogicalPlan] with Logging {
     val conf = new RapidsConf(plan.conf)
     // iterating over NamedExpression
     exp match {
-      case f: ScalaUDF => // found a ScalaUDF
+      case f: ScalaUDF if getRapidsUDFInstance(f.function).isEmpty =>
```

Comment:
Nit, could there be a comment above this stating why the check is required?

Reply:
Added.